ArcTouch.Core/Models/ApiModels/ResponseIMDBMoviesList.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ArcTouch.Core.Models.Api
+{
+    public class ResponseIMDBMoviesList
+    {
+        [JsonProperty(PropertyName = "page")]
```

Comment:
To avoid the attribute, you could use the  `SnakeCaseNamingStrategy`
https://www.newtonsoft.com/json/help/html/NamingStrategySnakeCase.htm

Reply:
Omg!!! very practical. I didn't know that. thanks for the tip.